lesson_bayes/Lesson_Bayes - Bayes Rule Intro.ipynb

```diff
@@ -0,0 +1,198 @@
+{
```

Comment:
<p>represents, I think there are spell checkers for JupyterNB.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>